scripts/releaser/release-debian.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# This script is interactive as you need to log in to 'az'.
+
+from __future__ import print_function
+
+import os
+import sys
+import time
+from datetime import datetime
+from six import StringIO
+from sh import az, ssh
+
+# Environment variables
+ENV_REPO_NAME = os.environ.get('REPO_NAME')
+ENV_CLI_VERSION = os.environ.get('CLI_VERSION')
+ENV_CLI_DOWNLOAD_SHA256 = os.environ.get('CLI_DOWNLOAD_SHA256')
+ENV_STORAGE_CONNECTION_STRING = os.environ.get('AZURE_STORAGE_CONNECTION_STRING')
+ENV_DEBIAN_REPO_ID = os.environ.get('DEBIAN_REPO_ID')
+ENV_DEBIAN_REPO_URL = os.environ.get('DEBIAN_REPO_URL')
+ENV_DEBIAN_REPO_USERNAME = os.environ.get('DEBIAN_REPO_USERNAME')
+ENV_DEBIAN_REPO_PASSWORD = os.environ.get('DEBIAN_REPO_PASSWORD')
+
+assert (ENV_REPO_NAME and ENV_CLI_VERSION and ENV_CLI_DOWNLOAD_SHA256 and ENV_STORAGE_CONNECTION_STRING and ENV_DEBIAN_REPO_ID and ENV_DEBIAN_REPO_URL and ENV_DEBIAN_REPO_USERNAME and ENV_DEBIAN_REPO_PASSWORD),\
+        "Not all required environment variables have been set. "\
+        "Set REPO_NAME, CLI_VERSION, CLI_DOWNLOAD_SHA256, AZURE_STORAGE_CONNECTION_STRING, DEBIAN_REPO_ID, DEBIAN_REPO_URL, DEBIAN_REPO_USERNAME, DEBIAN_REPO_PASSWORD"
```

Comment:
I want to do a check at the beginning before starting anything.
If the release script fails midway through because an env var wasn't set, that wouldn't be good.